composer.json

```diff
@@ -36,4 +35,4 @@
         "email": "support@uploadcare.com",
         "issues": "https://github.com/uploadcare/uploadcare-php/issues"
     }
-}
+}
```

Comment:
:(

Reply:
what's wrong here?

Reply:
no new line at EOF, contradicting your editorconfig

Reply:
yes, fixed in my last commit